core/Plugin/Menu.php

```diff
@@ -30,16 +30,6 @@
  */
 class Menu
 {
-    protected $module = '';
```

Comment:
I think @tsteur wrote the Menu class, so I'm not sure if this was his intent or not, but I think the idea for using a variable instead of getModule() directly was so derived classes could override the value. @tsteur can you confirm?